Topics/Tech_Stacks/FastAPI.md

```diff
@@ -0,0 +1,106 @@
+# FastAPI
+
+## Table of Contents
+### [Introduction](#introduction-1)
+### [Writing APIs](#writing-apis-1)
+### [Testing](#testing-1)
+
+
+## Introduction
+
+FastAPI is a popular web framework in Python for writing REST APIs. It is fast and easy to use, with type hints and automatic data validation. 
```

Comment:
I think that it could be helpful to list a couple of alternative Python web frameworks, as it may help the reader to get a better idea of what FastAPI is by making the connection to another web framework, such as Django or Flask.